apps/github-cascading-app/src/cascading/cascading.ts

```diff
@@ -251,9 +239,41 @@ export abstract class Cascading {
       .replaceAll('$target', targetBranch);
   }
 
+  /**
+   * Update the pull request message based on its context
+   * @param pullRequest Pull Request object
+   * @param context Context of the pull request
+   */
+  protected async updatePullRequestWithNewMessage(pullRequest: CascadingPullRequestInfo, context: PullRequestContext) {
+    this.logger.debug(`Updating the PR ${pullRequest.id}`);
+    const message = await this.generatePullRequestBody({
+      ...context,
+      bypassReviewers: false
+    }, pullRequest);
+
+    return this.updatePullRequestMessage(pullRequest.id, message);
+  }
+
+  /**
+   * Update the pull request message if the pull request can be merged
```

Comment:
Not sure about it does what your description says.

Reply:
missing a `not` indeed